### PR TITLE
Library page start

### PR DESCRIPTION
**Pull request by BrunoRosendo**

### CODE REVIEW

File: src/app_feup/lib/view/Widgets/search_filter_form.dart [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+              child: Text('Cancelar'), onPressed: () => Navigator.pop(context)),
+          ElevatedButton(
+              child: Text('Confirmar'),
+              onPressed: () {
+                // TODO Update state with filters
+                Navigator.pop(context);
+              })
+        ],
+        content: Container(
+          height: 350.0,
```
Comment (monkin77) [inline src/app_feup/lib/view/Widgets/search_filter_form.dart]: I'm not sure why it's not being done in the whole App, but I think that leaving the height and width in absolute value is not a great practice if we want to have a responsive design. I've created auxiliary functions to calculate the height and width in runtime, using a specific size as a reference.

Comment (monkin77) [reply src/app_feup/lib/view/Widgets/search_filter_form.dart]: I'll leave the changes in another pr.

Comment (BrunoRosendo) [reply src/app_feup/lib/view/Widgets/search_filter_form.dart]: True. I just did the same as somewhere else but it's pretty weird. Is the size in pixels? (as in 350px of height)

Comment (monkin77) [reply src/app_feup/lib/view/Widgets/search_filter_form.dart]: Yep, I think it is in pixels.

**Review by monkin77 — APPROVED** [review]
Looks good to me

**Pull request merged**